Library/Homebrew/cmd/test-bot.rb

```diff
@@ -445,6 +445,10 @@ def formula formula_name
         test "brew", "test", "--verbose", formula_name if formula.test_defined?
         testable_dependents.each do |dependent|
           unless dependent.installed?
+            conflicts = dependent.conlicts.map { |c| Formulary.factory(c.name) }.select { |f| f.installed? }
```

Comment:
Sadly, it doesn't. As shown in [CI bot](http://bot.brew.sh/job/Homebrew%20Pull%20Requests/19575/version=yosemite/console), this problem cannot be reproduced on Yosemite.

I'm searching for another formula to test this.